Sources/UniversalMailComposer.swift

```diff
@@ -39,30 +46,61 @@ open class UniversalMailComposer: NSObject, MFMailComposeViewControllerDelegate
         }
     }
     
-    func fallbackClients(to: String, subject: String, body: String, attachmentData: Data? = nil, mimeType: String? = nil, attachmentName: String? = nil) -> URL? {
-        guard let subjectEncoded = subject.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed), let bodyEncoded = body.addingPercentEncoding(withAllowedCharacters: .urlQueryAllowed) else { return URL(string: "") }
+    func fallbackClients(to: String?, subject: String?, body: String?, attachmentData: Data? = nil, mimeType: String? = nil, attachmentName: String? = nil) -> URL? {
```

Comment:
Proposta, perché in generale mi piacciono poco le variabili dal nome generico
```suggestion
    func fallbackClients(to recipients: String?, subject: String?, body: String?, attachmentData: Data? = nil, mimeType: String? = nil, attachmentName: String? = nil) -> URL? {
```